src/dotnet/commands/dotnet-pack/PackCommand.cs

```diff
@@ -17,37 +17,41 @@ public static int Run(string[] args)
             CommandLineApplication cmd = new CommandLineApplication(throwOnUnexpectedArg: false)
             {
                 Name = "pack",
-                FullName = "pack",
-                Description = "pack for msbuild",
+                FullName = LocalizableStrings.AppFullName,
+                Description = LocalizableStrings.AppDescription,
                 AllowArgumentSeparator = true,
                 ArgumentSeparatorHelpText = HelpMessageStrings.MSBuildAdditionalArgsHelpText
             };
 
             cmd.HelpOption("-h|--help");
 
-            var output = cmd.Option("-o|--output <OUTPUT_DIR>",
-                "Directory in which to place outputs",
+            var output = cmd.Option(
+                $"-o|--output <{LocalizableStrings.CmdOutputDir}>",
+                LocalizableStrings.CmdOutputDirDescription,
                 CommandOptionType.SingleValue);
             var noBuild = cmd.Option("--no-build",
-                "Do not build project before packing", 
+                LocalizableStrings.CmdNoBuildOptionDescription, 
                 CommandOptionType.NoValue);
             var includeSymbols = cmd.Option("--include-symbols",
-                "Include PDBs along with the DLLs in the output folder",
+                LocalizableStrings.CmdIncludeSymbolsDescription,
                 CommandOptionType.NoValue);
             var includeSource = cmd.Option("--include-source",
-                "Include PDBs and source files. Source files go into the src folder in the resulting nuget package",
+                LocalizableStrings.CmdIncludeSourceDescription,
                 CommandOptionType.NoValue);
-            var configuration = cmd.Option("-c|--configuration <CONFIGURATION>",
-                "Configuration under which to build", 
+            var configuration = cmd.Option(
+                $"-c|--configuration <{LocalizableStrings.CmdConfig}>",
+                LocalizableStrings.CmdConfigDescription, 
                 CommandOptionType.SingleValue);
-            var versionSuffix = cmd.Option("--version-suffix <VERSION_SUFFIX>",
-                "Defines what `*` should be replaced with in version field in project.json",
+            var versionSuffix = cmd.Option(
+                $"--version-suffix <{LocalizableStrings.CmdVersionSuffix}>",
+                LocalizableStrings.CmdVersionSuffixDescription,
                 CommandOptionType.SingleValue);
-            var serviceable = cmd.Option("-s|--serviceable", 
-                "Set the serviceable flag in the package", 
+            var serviceable = cmd.Option("-s|--serviceable",
```

Comment:
You moved some of the parameters to their own lines, but you left this one and a couple others. "-s|--serviceable" should also be in its own line. Do a pass and see other cases that may apply.